environment.yml

```diff
@@ -1,17 +1,14 @@
 # Simple install with
 # mamba env create -f environment.yml
-name: myria3d
+name: myria3d_latest_pytorch
```

Comment:
why do you change the environment name? Is it something you used in your tests and forgot to remove?

Reply:
Yes indeed! Will correct.